src/Pages/Pages/LoginPage/LoginPage.css

```diff
@@ -0,0 +1,81 @@
+.login-wrapper {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  margin-top: 6rem;
+}
+
+.login-page-bottom-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.login-container {
+  display: flex;
+  flex-direction: column;
+  background-color: var(--component-container-color);
+  width: 27rem;
+  display: flex;
+  justify-content: center;
+  border-radius: 5px;
+  align-items: center;
+}
+
+.login-form {
+  margin: 1rem;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.login-form:nth-child(5) {
+  justify-self: flex-start;
+}
+
+.form-input-container {
+  width: 350px;
```

Comment:
for big sizes you can use relative sizes

Reply:
use rem or %